Snake/SnakeTest/GameSnake_Test.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Linq;
+using System.Text;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace SnakeTest
+{
+    [TestClass]
+    public class GameSnake_Test
+    {
+        [TestMethod]
+
+        public void First_Snake_Created_at_Correct_Location()
+        {
+            //create new snake 
+            BusinessLayer.GameSnake testSnake = new BusinessLayer.GameSnake();
+            List<BusinessLayer.Point> testSnakeList;
+            testSnakeList = testSnake.createFirstSnake(40, 40, 3);
+
+
+            //Assert.IsTrue(testSnakeList.);
+            Assert.IsTrue(testSnakeList.Count()==3);
+            // Assert.IsTrue(testSnakeList.in)
+            Assert.IsTrue(testSnakeList.ElementAt(0).returnX() == 20);
+            Assert.IsTrue(testSnakeList.ElementAt(0).returnY() == 20);
+        }
+
+
+        [TestMethod]
+
+        public void Snake_Moves_Right_Correctly()

```

Comment:
Test name should be named - MethodBeingTested_Scenario_Result or MethodBeingTested_Result _Scenario, so you should use you comment in the test for example,

snakeMove_when_eating_food_snake_moves_right()